### DISCUSSION TITLE
umbraco 11 sharing back-office authenticated user session in a sub-domain

### DISCUSSION THREAD

**Discussion by mitraganesh** [umbraco/Umbraco-CMS#14540] [Features and ideas]
I have 3 domains in one umbraco instance
https://www.site1.com 
https://www.site2.com 
https://www.site3.com 

Added culture and hostname on node and everything working as expected
Now I want to carry back office user authentication in other domain.
Is that possible?

e.g.
If someone login with domain https://www.site1.com and trying to access backoffice via https://www.site2.com  and no login screen will come up.

**Comment by bergmania** [comment]
I think people usually  setup redirect rules, to redirect from https://www.site2.com/umbraco to https://www.site1.com/umbraco
